charts/lily/templates/configmap-daemonconfig.yaml

```diff
@@ -39,4 +39,17 @@ data:
           Path = {{ .path  | quote }}
         {{- end }}
       {{- end }}
+    [Queue]
+      {{- if .Values.daemon.queue.asynq }}
+      [Queue.Asynq]
+        {{- range .Values.daemon.queue.asynq }}
+        [Queue.Asynq.{{ .name }}]
+          Network = {{ .network | default "tcp" | quote }}
+          Addr = {{ .addr | quote }}
+          Username = {{ .username | default "default" | quote }}
+          Password = {{ .password | quote }}
```

Comment:
It's an antipattern to put secrets in plaintext (configMaps are plaintext, btw). This also requires the secret to be injected into the deployment pipeline as well which can be annoying.

This approach is crude, but for Lily I currently handle the secret creation in k8s manually, keeping the passwords external to the repo/deployments. Then configure all the  nodes to look at the info in the secret.

Secrets can be exposed in the pod via filesystem or via envvars. We can leave it this way, but I think it's a footgun for other operators. We don't want to encourage plaintext passwords.

I think the best way to solve this would be to change this value to some envvar name (like postgres storage). The chart would also have a value that contains the name of the secret that the redis credentials are in. On deploy, helm would manage associating the data in the secret found at the name provided in the values with the envvar inserted into the config. (The envvar doesn't need to be managed by the chart user, helm could just generate something that makes sense and handle everything internally.

This would force the user to put secrets in a safe place and makes lily more friendly to other non-k8s operators who prefer to configure via envvars. (And if someone really would rather edit the config, they could still do that by changing the envvar name directly and leaving the secret status in the env.